spec/factories/seats.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :seat do
+    code { [*('A'..'Z')].sample + %w[1 2].sample }
+    status { %i[seat_available on_hold booked].sample }
+    trip_id { create(:trip).id }
```

Comment:
don't you think something else can be done here instead of create each entity and then accessing it's ID?

Reply:
Apu, I used `trip { create(:trip) }` instead of what I did earlier and it worked! Kindly let me know if there are any better options.